shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -2250,4 +2253,12 @@ object GlutenConfig {
       .doc("If enabled, gluten will convert the viewfs path to hdfs path in scala side")
       .booleanConf
       .createWithDefault(false)
+
+  val VELOX_BROADCAST_BUILD_RELATION_USE_OFFHEAP =
+    buildConf("spark.gluten.velox.offHeapBroadcastBuildRelation.enabled")
+      .internal()
+      .doc("If enabled, broadcast build relation will use offheap memory. " +
+        "Otherwise, broadcast build relation will use onheap memory.")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
Let's default it to false, and need a smooth transition period?

Reply:
Yes, let's mark the doc with `(experimental)` prefix as well. Also mark `UnsafeBytesBufferArray` / `UnsafeColumnarBuildSideRelation` with `@Experimental` annotations.